modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -116,15 +115,15 @@
 	alt_titles = list("Psychologist", "Psychiatrist", "Therapist", "Counsellor")
 
 /datum/job/quartermaster
-	alt_titles = list("Quartermaster", "Deck Chief", "Cargo Foreman")
+	alt_titles = list("Quartermaster", "Deck Chief", "Cargo Foreman", "Head of Cargo", "Chief Transport Officer")
```

Comment:
I dont see why the Supply head-of-staff is being referred to as Transportation Officer rather than Requisition, Supply, Cargo, or Shipping Officer. Transport makes it feel like they have control over all shuttles IMO

Reply:
they transport cargo back and forth

and, it initializes to CTO, apposed to CSO, which is the science officer

and chief transport sounds better than chief cargo

Reply:
Chief Requisition Officer sounds better
or Chief Financial Officer

Reply:
i like CRO yeah